src/routes/adminRoutes.js

```diff
@@ -2,8 +2,10 @@ const express = require('express');
 const router = express.Router();
 const expressBasicAuth = require('express-basic-auth');
 const adminController = require('../controllers/adminController');
-
-router.get('/', adminController.adminHomePage);
```

Comment:
Did this get moved somewhere else? I'm not able to load the admin page without this (I get a "Route does not exist" page)

Reply:
Sorry, accidentally.